src/main/java/com/microfocus/octane/plugins/configuration/ConfigurationUtil.java

```diff
@@ -385,4 +385,16 @@ private static void validateJiraIssuesList(WorkspaceConfigurationOutgoing wco) {
 
         return usedJiraProjects;
     }
+
+    public static String getSpaceConfigurationIdByBaseurl(String baseurl, long spaceId) {
```

Comment:
I don't think you need this method, but if you do, I'd rename baseurl to baseUrl both as parameter and as method name